rules/rollup.go

```diff
@@ -203,3 +213,63 @@ func bytesArrayCopy(values [][]byte) [][]byte {
 	}
 	return result
 }
+
+func (t rollupTarget) Schema() (*schema.RollupTarget, error) {
+	res := &schema.RollupTarget{
+		Name: string(t.Name),
+	}
+
+	policies := make([]*schema.Policy, len(t.Policies))
+	for i, p := range t.Policies {
+		policy, err := p.Schema()
+		if err != nil {
+			return nil, err
+		}
+		policies[i] = policy
+	}
+	res.Policies = policies
+	res.Tags = stringArrayFromBytesArray(t.Tags)
+
+	return res, nil
+}
+
+// Schema returns the given MappingRuleSnapshot in protobuf form.
+func (rrs rollupRuleSnapshot) Schema() (*schema.RollupRuleSnapshot, error) {
```

Comment:
Can we move the methods closer to where the other methods are defined for this type? Makes it easier to read/find stuff.